src/services/fields.ts

```diff
@@ -118,7 +119,8 @@ export function getFilterBreakdownValueScene(
   getTitle: (df: DataFrame) => string,
   style: DrawStyle,
   variableName: typeof VAR_FIELDS | typeof VAR_LABELS | typeof VAR_METADATA,
-  sortByScene: SortByScene
+  sortByScene: SortByScene,
+  labelKey?: string
```

Comment:
Are we setting this anywhere? At least in this PR I don't see it.

Reply:
Should be these changes: https://github.com/grafana/explore-logs/pull/840/files#diff-e7101021dce31dbf3cbbcb5f789bf846d662a1a45562ca278dd01b3703f3ecb5

Reply:
Perfect, thanks!